common/armies/hfe_armies.txt

```diff
@@ -38,3 +38,304 @@
 # 		}
 # 	}
 # }
+
+hfe_army_high_mechanicus_bodyguards = {
+	defensive = yes
+	is_pop_spawned = no
+	pop_limited = no
+	health = 1.1
+	damage = 2
+	morale = 5
+	has_morale = yes
+	morale_damage = 1.5
+	collateral_damage = 0.9
+	war_exhaustion = 2
+	icon_frame = 5
+	has_species = yes
+
+	resources = {
+		category = armies
+		cost = {
+#			trigger = {
```

Comment:
Should you remove unused stuff, it will make reviewing easier.

Reply:
No need.

Reply:
Do I really have to do that? I don't want to do it.